x-pack/plugins/security_solution/public/resolver/view/panels/process_cube_icon.tsx

```diff
@@ -13,13 +13,11 @@ import { useResolverTheme } from '../assets';
  */
 export const CubeForProcess = memo(function CubeForProcess({

```

Comment:
:grey_question: Not related to this PR, but I'm having trouble understanding why this file isn't part of `assets`